common/ext/class.Namespace.php

```diff
@@ -18,148 +18,100 @@
  *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
  *
  */
-
+
 /**
  * Short description of class common_ext_Namespace
  *
- * @access public
- * @author Jerome Bogaerts, <jerome.bogaerts@tudor.lu>
+ * @access  public
+ * @author  Jerome Bogaerts, <jerome.bogaerts@tudor.lu>
  * @package generis
-
  */
 class common_ext_Namespace
 {
-
-    // --- ASSOCIATIONS ---
-    // generateAssociationEnd :
-
-    // --- ATTRIBUTES ---
-
     /**
      * A unique identifier of the namespace
      *
-     * @access protected
-     * @var int
+     * @var string
      */
-    protected $modelId = 0;
-
+    protected $modelId;
+
     /**
      * the namespace URI
      *
-     * @access protected
      * @var string
      */
-    protected $uri = '';
-
-    // --- OPERATIONS ---
-
+    protected $uri;
+
     /**
-     * Create a namespace instance
+     * Namespace constructor.
      *
-     * @access public
      * @author Jerome Bogaerts, <jerome.bogaerts@tudor.lu>
-     * @param  int id
+     *
+     * @param  string id
      * @param  string uri
-     * @return mixed
      */
-    public function __construct($id = 0, $uri = '')
+    public function __construct($id = '', $uri = '')
```

Comment:
Can be changed to
`public function __construct($id = '', $uri = '')`
with no problem.